src/elements/element.line.js

```diff
@@ -1,12 +1,17 @@
 'use strict';
 
 import defaults from '../core/core.defaults';
-import Element from '../core/core.element';
+import ChartElement from '../core/core.element';
```

Comment:
Why this change?  `Arc` and `Scale` use `Element` still.

Reply:
I was getting some errors after the latest rebase and thought that perhaps TypeScript was getting confused with an HTML Element. It looks like this change isn't necessary, so I can revert it